ads/google/a4a/utils.js

```diff
@@ -293,3 +309,47 @@ export function getCorrelator(win, opt_cid) {
   }
   return win.ampAdPageCorrelator;
 }
+
+/**
+ * Creates or reinitializes a lifecycle reporter for Google ad network
+ * implementations.
+ *
+ * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4aElement
+ * @return {!./performance.GoogleAdLifecycleReporter}
+ */
+export function googleLifecycleReporterFactory(a4aElement) {
+  const reporter =
+      /** @type {!./performance.GoogleAdLifecycleReporter} */
+      (getLifecycleReporter(a4aElement, 'a4a', undefined,
```

Comment:
Good catch.  Looks like the corr was added in a previous PR, but never really used.  I'll check with Gene and see if he has some vision for the arg.  Otherwise, I'm happy to pull it out.